contributing.md

```diff
@@ -48,69 +40,136 @@ See the License for the specific language governing permissions and
+
+```
+$ANDROID_SDK_ROOT/tools/bin/sdkmanager "system-images;android-27;google_apis;x86"
+$ANDROID_SDK_ROOT/tools/bin/sdkmanager "system-images;android-21;google_apis;x86"
+```
+Create Android Virtual Devices (AVDs):
+
+```
+$ANDROID_SDK_ROOT/tools/bin/avdmanager create avd --name "bpk-avd" --package "system-images;android-27;google_apis;x86" --device "pixel" && cp native/android/bpk-avd.ini ~/.android/avd/bpk-avd.avd/config.ini
+$ANDROID_SDK_ROOT/tools/bin/avdmanager create avd --name "bpk-avd-min" --package "system-images;android-21;google_apis;x86" --device "Nexus 5"
+```
+</details>
 
-See the [decisions directory](/decisions) for an organised list of cultural and code decisions
-made concerning this monorepo.
+### Code style
+
+Backpack uses a combination of [ESLint](https://eslint.org) and [Prettier](https://prettier.io) to enforce coding styles. ESLint runs as a pre-commit hook, so it isn't possible to commit code that causes ESLint to fail.
+
+We recommend that you install [a plugin to your editor](https://eslint.org/docs/user-guide/integrations#editors) to run ESLint automatically, which will then show you any errors inline. You can even enable an option to fix ESLint errors automatically upon saving.
+
+## Getting started
+
+### Install dependencies
+
+Run `npm install` to install dependencies from npm.
+
+<details>
+<summary>A note on dependencies</summary>
+
+Backpack is a multi-package repository, also known as a monorepo. This means that instead of having one code repository for each npm package, we manage them all inside this single repository.
+
+We use [Lerna](https://lernajs.io) to achieve this. Lerna links packages together inside this repo by 'bootstrapping'. 
+
+When you run `npm install`, Lerna is bootstrapped automatically as a post-install hook. However, if you change dependencies within a package, you will need to run Lerna manually with `npm run bootstrap`.
+
+</details>
+
+#### React Native
+
+> Skip this section if you don't intend to develop React Native components.
+
+<details>
+<summary>iOS</summary>
+
+From inside `native/ios`, run `bundler install` followed by `bundle exec pod install`.
+</details>
+
+<details>
+<summary>Android</summary>
+
+To ensure that maps powered by Google work set the `google_maps_api_key` in `native/android/local.properties` and make sure you are using the backpack.keystore.
+
+##### APK signing
+
+For members of Backpack we have a keystore tied to our Google Maps API key in LastPass. Retrieve this key and place it in `native/android/backpack.keystore`. For contributors who are not members of Backpack nothing needs to be done, but Google Maps will not work. If you need Google Maps to work you'll need to supply your own Google Maps Api Key and possible keystore.
+
+</details>
+
+### Build the code
+
+Build the code with `npm run build`.
```

Comment:
I think this is misleading - we are not building the code, we are build run time dependencies for our javascript code. These dependencies are: tokens, svgs, fonts etc.

Reply:
Yeah maybe `build the code` is an oversimplification.